### PR TITLE
Added code generation for `UnitIdentifier`

### LINKED ISSUE #59: Generate unit tests and extensions for `UnitIdentifier`
The defined Bluetooth Units should be specified in a list and automatically generate the extensions and unit tests. The member names should be manually defined and not sanitized from the documentation.

### PR DESCRIPTION
**Pull request by carlos21**
**Issue**

Fixes #59 .

**What does this PR Do?**

Added generated code for units.

**Where should the reviewer start?**

DarwinTests.swift

### CODE REVIEW

**Comment by colemancda** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/af8ba83602f5b971de75e8fa84e7feb9/af8ba83602f5b971de75e8fa84e80b1b)
---
*Duration*: 18 minutes and 0 second
*Result*: **Perfect build!** All 151 tests passed. :+1:
*Test Coverage*: 85%

File: Tests/BluetoothTests/DarwinTests.swift [hunk 1]
```diff
@@ -357,6 +357,240 @@ final class DarwinTests: XCTestCase {
...
+        🖨("//")
+        🖨("//  Generated by Carlos Duclos on \(fileDate).")
+        🖨("//")
+        🖨("")
+        🖨("public extension UnitIdentifier {")
+        🖨("")
+        
+        for (identifier, unit) in units {
+            
+            var memberName = unitsMethodNames[identifier]!
```
Comment (colemancda) [inline Tests/BluetoothTests/DarwinTests.swift]: should use `guard`

File: Tests/BluetoothTests/DarwinTests.swift [hunk 2]
```diff
@@ -357,6 +357,240 @@ final class DarwinTests: XCTestCase {
...
+            let hexValue = "0x\(identifier.toHexadecimal())"
+            let comment = unit.name + " " + "(`\(hexValue)`)"
+            
+            🖨("    /// " + comment)
+            🖨("    static var " + memberName + ": UnitIdentifier {")
+            🖨("        return UnitIdentifier(rawValue: \(hexValue))")
+            🖨("    }")
+            🖨("")
+            
+            memberNameCache[identifier] = memberName
```
Comment (colemancda) [inline Tests/BluetoothTests/DarwinTests.swift]: `memberNameCache == unitsMethodNames` so `memberNameCache` should be removed

**Comment by colemancda** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/af8ba83602f5b971de75e8fa84e7feb9/af8ba83602f5b971de75e8fa84e80b1b)
---
*Duration*: 18 minutes and 0 second
*Result*: **Perfect build!** All 151 tests passed. :+1:
*Test Coverage*: 85%

File: Tests/BluetoothTests/DarwinTests.swift (hunk 2 above)
Comment (carlos21) [reply Tests/BluetoothTests/DarwinTests.swift]: @colemancda I'm using memberNameCache to validate duplicates, so memberNameCache != unitsMethodNames xD

**Pull request merged**